spec/amp-var-substitutions.md

```diff
@@ -546,6 +548,22 @@ Provides the absolute Y coordinate of the top edge of the element specified by `
 * **platform variable**: N/A
 * **amp-analytics variable**: `${elementY}`
 
+#### Error Message
+
+Provides the message of the error that triggered an user error event. This variable is only available in a `trigger` of type `user-error`
+
+* **platform variable**: N/A
+* **amp-analytics variable**: `${errorMessage}`
+  * Example: `"error": "http://localhost:8000/message=${errorMessage}"`
```

Comment:
Example would be something like an expected string value for an error.

Reply:
Also, nit: Should say "Example value" to be consistent with other entries on the page.